packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -17,13 +17,23 @@ const GuidanceWrapper = styled.div`
   height: 100%;
   position: absolute;
   background-color: ${GUIDANCE_BACKGROUND};
+  border: 1px solid transparent;
```

Comment:
Does this 1px transparent border need to exist when HCM is _not_ active? It creates additional padding which skews the GEL values.

Reply:
This transparency fix isn't for HCM, this a fix for firefox's colour preferences which cannot be targeted with a media query. Without the border a user with colour preferences won't be able to distinguish where the video is:
<img width="710" alt="Screenshot 2019-11-11 at 11 02 37" src="https://user-images.githubusercontent.com/4706195/68582566-f61b7980-0472-11ea-8d07-cf3445e115e4.png">
<img width="710" alt="Screenshot 2019-11-11 at 11 01 50" src="https://user-images.githubusercontent.com/4706195/68582577-fc115a80-0472-11ea-8905-762b4b2f8a9b.png">

Reply:
Cool, ok! Can we use `0.0625rem` instead of `1px` here, please?